README.md

```diff
@@ -1,7 +1,47 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Mhammed Doukkali . Jarvis Consulting
+
+As a recent graduate from Concordia University, I bring a strong technical foundation to my five years of experience in the banking industry. My excellent track record in collaborating with team members, coupled with my ability to proactively address the needs of both customers and the organization, has honed my communication and problem-solving skills. My passion for software engineering began while working on a project to improve the online banking application. Intrigued by the power of programming to create and build solutions, I decided to enroll in an 'Introduction to Programming' course. Quickly, I found programming to be highly accessible and was thrilled to see the immediate results of my work in just a few lines of code. Currently, as an Applications and Data Engineer at Jarvis, I have honed my proficiency in various technologies such as Java, Linux, and Docker. I have demonstrated my ability to work in a fast-paced environment and deliver high-quality solutions. I am eager to leverage my skills and knowledge to contribute to your team's succes.
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git
+
+**Competent:** Javascript, ReactJS, HTML/CSS, MongoDB, REST API
+
+**Familiar:** Docker, Kotlin, Android Studio, NodeJS, Styled components
+
+## Jarvis Projects
+
+Project source code: [https://github.com/Jarvis-Consulting-Group/jarvis_data_eng-MhammedDoukkali/](https://github.com/Jarvis-Consulting-Group/jarvis_data_eng-MhammedDoukkali/)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/Jarvis-Consulting-Group/jarvis_data_eng-MhammedDoukkali//tree/masterhttps://github.com/Jarvis-Consulting-Group/jarvis_data_eng-MhammedDoukkali/linux_sql)]: Implemented a monitoring agent in Bash that records the hardware specifications and usage of host machines over time and saves the data to a Postgres database running in a Docker container. The testing process involved using a CentOS 7 virtual machine on the Google Cloud Platform, and I manually tested each script to confirm that it meets its specifications.
```

Comment:
for the last sentence you can use the phrase "and each script was manually tested to confirm that output expectations were met" as opposed to "and I manually tested each script to confirm that it meets its specifications."